### PR TITLE
FISH-7522 Payara Community 6.2023.7 Release Notes and TCK Results

### PR DESCRIPTION
**Pull request by abdulrahim458**
FISH-7522 Payara Community 6.2023.7 Release Notes and TCK Results

### CODE REVIEW

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+= MicroProfile 6.0 Certification Summaries
+
+Payara Server is a certified https://projects.eclipse.org/projects/technology.microprofile/releases/6.0[MicroProfile 6.0] implementation.
+As proof of compatibility, the output of the MicroProfile TCK Test Suite tests are presented for each certified version.
+
+The following distributions of Payara 6.2023.6 are certified, with results for each listed below:
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc:6]:
```suggestion
The following distributions of Payara 6.2023.7 are certified, with results for each listed below:
```

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+* xref:Eclipse MicroProfile Certification/6.2023.6/Server TCK Results.adoc[Payara Server 6.2023.6]
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc:8]:
```suggestion
* xref:Eclipse MicroProfile Certification/6.2023.7/Server TCK Results.adoc[Payara Server 6.2023.7]
```

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+* xref:Eclipse MicroProfile Certification/6.2023.6/Server Web TCK Results.adoc[Payara Server 6.2023.6 - Web profile]
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc:9]:
```suggestion
* xref:Eclipse MicroProfile Certification/6.2023.7/Server Web TCK Results.adoc[Payara Server 6.2023.7 - Web profile]
```

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Server TCK Results.adoc [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+https://download.eclipse.org/microprofile/microprofile-jwt-auth-2.1/microprofile-jwt-auth-spec-2.1.html[MicroProfile JWT Auth 2.1]
+https://download.eclipse.org/microprofile/microprofile-metrics-5.0.0/microprofile-metrics-spec-5.0.0.html[MicroProfile Metrics 5.0.0]
+https://download.eclipse.org/microprofile/microprofile-open-api-3.1/microprofile-openapi-spec-3.1.html[MicroProfile OpenAPI 3.1]
+https://download.eclipse.org/microprofile/microprofile-opentracing-3.0/microprofile-opentracing-spec-3.0.html[MicroProfile OpenTracing 3.0]
+https://download.eclipse.org/microprofile/microprofile-rest-client-3.0/microprofile-rest-client-spec-3.0.html[MicroProfile Rest Client 3.0]
+https://download.eclipse.org/microprofile/microprofile-telemetry-1.0/tracing/microprofile-telemetry-tracing-spec-1.0.html[MicroProfile Telemetry Tracing 1.0]
+
+- Public URL of TCK Results Summary:
++
+https://docs.payara.fish/community/docs/6.2023.6/Eclipse%20MicroProfile%20Certification/6.2023.6/Server%20TCK%20Results.html
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Server TCK Results.adoc:32]:
```suggestion
https://docs.payara.fish/community/docs/6.2023.7/Eclipse%20MicroProfile%20Certification/6.2023.7/Server%20TCK%20Results.html
```

File: community/docs/modules/ROOT/pages/Release Notes/Release Notes 6.2023.7.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+* Jakarta EE 10 Applications
+* MicroProfile 6.0
+
+== Known Issue
+=== Hazelcast Warning
+* The Hazelcast 5.3.1 upgrade adds socket options for per-socket keep-alive configuration. When running Payara on Windows, this version of Hazelcast throws a warning in the server logs, similar to this: "Ignoring TCP_KEEPCOUNT. It seems your JDK does not support jdk.net.ExtendedSocketOptions on this OS. Try upgrading to the latest JDK or check with your JDK vendor.Alternatively, on Linux, configure tcp_keepalive_probes in the kernel (affecting default keep-alive configuration for all sockets): For more info see [https://tldp.org/HOWTO/html_single/TCP-Keepalive-HOWTO/[keep alive]]. If this isn't dealt with, idle connections could be closed prematurely." This is a Hazelcast known issue that is documented [https://docs.hazelcast.com/hazelcast/5.3/clusters/network-configuration#configuring-tcp-keep-alive[here]].
+
+== Improvements
+
+* [https://github.com/payara/Payara/pull/6292[FISH-1419]] [Community Contribution - https://github.com/lprimak[Iprimak]] Hazelcast Partition Group according to Instance Group
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Release Notes/Release Notes 6.2023.7.adoc:15]:
```suggestion
* [https://github.com/payara/Payara/pull/6292[FISH-1419]] [Community Contribution - https://github.com/lprimak[lprimak]] Hazelcast Partition Group according to Instance Group
```

**Review by JamesHillyard — CHANGES_REQUESTED** [review]

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc (hunk 1 above)
Comment (abdulrahim458) [reply community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc]: Corrected the version

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc (hunk 2 above)
Comment (abdulrahim458) [reply community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Overview.adoc]: Corrected the version

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Server TCK Results.adoc [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+- Product Name, Version and download URL (if applicable):
++
+[cols="1,2",grid=none,frame=none]
+|===
+|image:JakartaEE_Logo_compatible-color.png[]
+|
+{empty} +
+{empty} +
+https://www.payara.fish/downloads/payara-platform-community-edition/[Payara Server 6.2023.6]
```
[the remark this answers is not in the document]
Comment (abdulrahim458) [reply community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Server TCK Results.adoc]: Corrected the version

File: community/docs/modules/ROOT/pages/Release Notes/Release Notes 6.2023.7.adoc (hunk 5 above)
Comment (abdulrahim458) [reply community/docs/modules/ROOT/pages/Release Notes/Release Notes 6.2023.7.adoc]: Corrected the url

File: community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Server TCK Results.adoc [hunk 7]
```diff
@@ -0,0 +1,120 @@
+[[tck-results]]
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile 6.0.
+
+**Payara Server 6.2023.6 Full Profile Certification Summary**
```
[the remark this answers is not in the document]
Comment (abdulrahim458) [reply community/docs/modules/ROOT/pages/Eclipse MicroProfile Certification/6.2023.7/Server TCK Results.adoc]: Corrected the version

**Review by JamesHillyard — APPROVED** [review]

**Pull request merged**